Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/FormatterStatistics.kt

```diff
@@ -33,7 +33,7 @@ fun StatsItem.getPrimaryLabel(context: Context): String {
             yesterday -> context.getString(R.string.statistics_primary_value_yesterday)
             else -> dateTimeFormatter.print(updatedAtDate)
         }
-        is IncidenceStats -> when (updatedAtDate) {
+        is IncidenceStats, is LocalIncidenceStats -> when (updatedAtDate) {
```

Comment:
Do you see a way to make this nicer in a future PR?
I don't like that we keep increasing our use of "formatters", was happy to remove the old ones we had.
Maybe just maybe some kind of abstract `StatsItem`?

For another PR ofc, just  thinking out loud here :grin:

Reply:
Sounds good, might be hard with more dynamic data but ill keep it in mind for after the RC